buildspec.yml

```diff
@@ -33,6 +33,7 @@ phases:
       - docker-compose build
       - echo Running tests...
       - scripts/lint.sh
+      - scripts/django-checks.sh
```

Comment:
Does this need to go after the check-migrations script? @sztamas

Reply:
I don't think it matters. It's fine there.